### PR TITLE
docs(beginner-guide): starting the aircraft

### PR DESCRIPTION
**Pull request by scoopscode**
Made a new PR after GitHub Desktop acting up

### CODE REVIEW

**Comment by frankkopp** [comment]
General feedback regarding content: 

- I like that you used "we" form - I did as well :)
- Use the FlyByWire checklist in the chapters - e.g. "Before Start"
- Let's assume Ext power and add a info box for when not available - should say turn on APU now 
- Are you sure the parking brake should be OFF (pre-checks)?
- Fire Test: Also look at ECAMs for the fire wqarning messages
- Fire Test before Fuel Pumps
- APU missing (see engine-start-taxi.md prerequisits - these need to match)
- Missing section about lighting (cabin - also displays but you have that already)
- Maybe add to contact ATC to get clearance for the flightplan at the end 
- You could in general also describe the Overhead Scan Method (https://youtu.be/uqKeSO6msDk?t=128)

Let me know if you need help with the images.

File: docs/beginner-guide/starting-the-aircraft.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+## Chapters / Phases
+
+This guide will cover these chapters in order:
+
+1. [Overhead Panel](#overhead-panel)
+2. [Main Panel](#main-panel)
+
+
+## Pre-Requisites
+To start off, we will begin by doing a few prechecks in all areas of the cockpit to ensure the aircraft is in an normal position before turning on the batteries.
```
Comment (frankkopp) [inline docs/beginner-guide/starting-the-aircraft.md]: pre-checks

File: docs/beginner-guide/starting-the-aircraft.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+After powering the aircraft, turn on the crew oxygen supply by pushing on the button, diminishing the white `OFF` light.
+
+To align the Inertial Reference System (or IRS), turn the 3 knobs to the `NAV` position. This will then start the IRS alignment which you can view the progress on the ECAM display.
+(IRS overhead image)(ECAM IRS progress image)
+
+To indicate that the aircraft is powered from the exterior, turn on the `NAV & LOGO` light. 
+Following, set the `STROBE` light to the `AUTO` position.   
+(Image)
+
+Once we our exterior lighting is in the correct state, we can then turn on the **seat belts sign and no smoking sign**.
```
Comment (frankkopp) [inline docs/beginner-guide/starting-the-aircraft.md]: remove "we"

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/beginner-guide/starting-the-aircraft.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+---
+
+## Overhead Panel
+
+Direct your attention to the **ELEC PANEL**
+
+Begin by, turning on batteries `1 AND 2`. After pushing these buttons, check to see on both sides that there is voltage.
+
+!!! info "External Power availible"
```
Comment (frankkopp) [inline docs/beginner-guide/starting-the-aircraft.md]: typo: available

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: docs/beginner-guide/starting-the-aircraft.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
+# Starting the Aircraft
+
+This guide will assist you with starting your aircraft. This guide includes images as well to allow you to understand the locations of all buttons and switches.
```
Comment (Valastiri) [inline docs/beginner-guide/starting-the-aircraft.md]:
```suggestion
This guide will assist you with starting your aircraft. It includes images to assist you with understanding the locations of all buttons and switches.
```
Double use of "this guide" is verbose.

File: docs/beginner-guide/starting-the-aircraft.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+!!! warning "Disclaimer"
+    The level of detail in this guide is meant to get a FlyByWire A320neo beginner starting the aircraft correctly.
```
Comment (Valastiri) [inline docs/beginner-guide/starting-the-aircraft.md]:
```suggestion
    The level of detail in this guide is meant to teach a FlyByWire A320neo beginner to start the aircraft correctly.
```

**Review by Valastiri — CHANGES_REQUESTED** [review]
Additional changes

**Comment by Valastiri** [comment]
> * Are you sure the parking brake should be OFF (pre-checks)?

This was debated in our ground school series. I think we can write this off as SOP and possibly since "MSFS" do this as parking brake ON instead.

**Review by Valastiri — APPROVED** [review]

**Review by frankkopp — APPROVED** [review]

**Comment by Valastiri** [comment]
Reviewed by Tarek and Shomas.

**Pull request merged**